src/routes/healthcheck/healthcheck-get.ts

```diff
@@ -1,8 +1,20 @@
 import { Request, Response } from "express";
 import { getLogger } from "config/logger";
+import { EncryptionClient } from "utils/encryption-client";
 
 const logger = getLogger("healthcheck");
-export const HealthcheckGet = (_: Request, res: Response): void => {
+export const HealthcheckGet = async (_: Request, res: Response): Promise<void> => {
```

Comment:
We have `/deepcheck` endpoint as well I wonder if that might be a better place for it.
Only because `/healthcheck` has always been super simple, just returning a 200?